instance/models/load_balancer.py

```diff
@@ -49,7 +48,7 @@ def _create_default(self):
         Create the default load balancing server configured in the Django settings, if any.
         """
         if settings.DEFAULT_LOAD_BALANCING_SERVER:
-            ssh_username, unused, domain = settings.DEFAULT_LOAD_BALANCING_SERVER.partition("@")
+            ssh_username, _, domain = settings.DEFAULT_LOAD_BALANCING_SERVER.partition("@")
```

Comment:
Yeah, I get your opinion on this, and I'm personally neutral; `_` is just something I got used to from working on edx-enterprise, but leaving it `unused` sounds good to me!

Reply:
@UmanShahzad Yeah, I realize that this is kind of moot, since I'm not going to stop this pattern from being used, but I just needed to take the opportunity to rant about it.